apps/system/js/sim_lock_system_dialog.js

```diff
@@ -167,10 +174,13 @@
     this.errorMsgBody = document.getElementById('messageBody');
 
     this.containerDiv = document.querySelector('#simlock-dialog .container');
+    this.dialogDone = document.querySelector('#simlock-dialog button.submit');
+    this.dialogSkip = document.querySelector('#simlock-dialog button.cancel');
```

Comment:
nit: I'd replace all `document.querySelector('#simlock-dialog selector')` by `this.element.querySelector('selector')`.
It's easier to read and slightly more efficient.